### PR TITLE
add mandarin doc

### PR DESCRIPTION
**Pull request by Slyne**
Add *docs_zh* under docs.
*docs_zh* is the directory of mandarin translation source code.

As usual, in order to get the html files:
```bash
cd docs/sources;
./update_docs.sh
```
there will be a *zh* folder under the *html* folder.

Notice we didn't use locale feature (generating the pot files &.po files), since it did not allow us to translate the comments in the code.

### CODE REVIEW

**Comment by okuchaiev** [comment]
when I do ":~/repos/NeMo/docs/sources$ ./update_docs.sh" I get the following error:

"""
... 

copying static files... ... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 17 warnings.

The HTML pages are in build/html.

Build finished. The HTML pages are in build/html.
rm -rf build/*
begin to generate mandarin html
rm -rf build/*
sphinx-build -b html -d build/doctrees   source build/html
Running Sphinx v2.2.1
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 35 source files that are out of date
updating environment: [new config] 35 added, 0 changed, 0 removed
reading sources... [  2%] api-docs/modules
reading sources... [  5%] api-docs/nemo

Exception occurred:
  File "/home/okuchaiev/repos/NeMo/docs/docs_zh/sources/source/conf.py", line 210, in process_docstring
    if isinstance(obj, type) and issubclass(obj, nemo.core.NeuralModule):
NameError: name 'nemo' is not defined
The full traceback has been saved in /tmp/sphinx-err-wqqnwytj.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Makefile:56: recipe for target 'html' failed
make: *** [html] Error 2
rm -rf build/*
Finish
"""
The same command finished successfully on master. Any ideas?

File: docs/docs_zh/sources/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+author = "NVIDIA"
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+# The short X.Y version.
+version = "0.8.1"
+# The full version, including alpha/beta/rc tags.
+release = "0.8.1"
```
[the remark this answers is not in the document]
Comment (okuchaiev) [reply docs/docs_zh/sources/source/conf.py]: please fix this to 0.9.0

File: docs/sources/source/nlp/bert_pretraining.rst [hunk 2]
```diff
@@ -106,73 +87,46 @@ We also need to define the BERT model that we will be pre-training. Here, you ca
...
+                                                     num_layers=2,
+                                                     log_softmax=True)
         nsp_loss_fn = nemo.backends.pytorch.common.CrossEntropyLoss()
 
         bert_loss = nemo_nlp.LossAggregatorNM(num_inputs=2)
 
-Then, we create the pipeline from input to output that can be used for both training and evaluation:
-
-For training from raw text use nemo_nlp.BertPretrainingDataLayer, for preprocessed data use nemo_nlp.BertPretrainingPreprocessedDataLayer
+Then, we create the pipeline gtom input to output that can be used for both training and evaluation:
```
[the remark this answers is not in the document]
Comment (Slyne) [reply docs/sources/source/nlp/bert_pretraining.rst]: @qijiaxing

File: docs/sources/source/nlp/bert_pretraining.rst [hunk 3]
```diff
@@ -1,23 +1,20 @@
 Pretraining BERT
```
[the remark this answers is not in the document]
Comment (Slyne) [reply docs/sources/source/nlp/bert_pretraining.rst]: @qijiaxing

File: docs/sources/source/index.rst [hunk 4]
```diff
@@ -56,70 +56,24 @@ See this video for a walk-through.
...
-    # Install the NLP collection from collections/nemo_nlp
-    cd NeMo/collections/nemo_nlp
-    pip install .
+    * NLP collection from `collections/nemo_nlp`: ``python setup.py install``
+4) For development you will need to: ``python setup.py develop`` instead of ``python setup.py install`` in Step (3.2) above
+5) Go to `examples/start_here` to get started with few simple examples
 
-    # Install the TTS collection from collections/nemo_tts
-    cd NeMo/collections/nemo_tts
-    pip install .
```
[the remark this answers is not in the document]
Comment (Slyne) [reply docs/sources/source/index.rst]: @MuyangDu

File: docs/docs_zh/sources/source/asr/tutorial.rst [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+
+    # 初始化神经模块
+    data_layer = nemo_asr.AudioToTextDataLayer(
+        manifest_filepath=train_dataset,
+        labels=labels, batch_size=32)
+    data_layer_val = nemo_asr.AudioToTextDataLayer(
+        manifest_filepath=eval_datasets,
+        labels=labels, batch_size=32, shuffle=False)
+
+    data_preprocessor = nemo_asr.AudioPreprocessing()
```
[the remark this answers is not in the document]
Comment (Slyne) [reply docs/docs_zh/sources/source/asr/tutorial.rst]: Sorry, before we translated this version, it is still the original one. I will take a look at the version difference and update our document to the latest one.

File: docs/sources/source/nlp/bert_pretraining.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (qijiaxing) [reply docs/sources/source/nlp/bert_pretraining.rst]: This error is solved now as we updated this file to align with latest version.

File: docs/sources/source/nlp/bert_pretraining.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (qijiaxing) [reply docs/sources/source/nlp/bert_pretraining.rst]: This was indeed a mistake during merging. Now we have update this doc file to align with the latest version.

**Comment by Slyne** [comment]
> when I do ":~/repos/NeMo/docs/sources$ ./update_docs.sh" I get the following error:
> 
> """
> ...
> 
> copying static files... ... done
> copying extra files... done
> dumping search index in English (code: en)... done
> dumping object inventory... done
> build succeeded, 17 warnings.
> 
> The HTML pages are in build/html.
> 
> Build finished. The HTML pages are in build/html.
> rm -rf build/*
> begin to generate mandarin html
> rm -rf build/*
> sphinx-build -b html -d build/doctrees source build/html
> Running Sphinx v2.2.1
> building [mo]: targets for 0 po files that are out of date
> building [html]: targets for 35 source files that are out of date
> updating environment: [new config] 35 added, 0 changed, 0 removed
> reading sources... [ 2%] api-docs/modules
> reading sources... [ 5%] api-docs/nemo
> 
> Exception occurred:
> File "/home/okuchaiev/repos/NeMo/docs/docs_zh/sources/source/conf.py", line 210, in process_docstring
> if isinstance(obj, type) and issubclass(obj, nemo.core.NeuralModule):
> NameError: name 'nemo' is not defined
> The full traceback has been saved in /tmp/sphinx-err-wqqnwytj.log, if you want to report the issue to the developers.
> Please also report this if it was a user error, so that a better error message can be provided next time.
> A bug report can be filed in the tracker at https://github.com/sphinx-doc/sphinx/issues. Thanks!
> Makefile:56: recipe for target 'html' failed
> make: *** [html] Error 2
> rm -rf build/*
> Finish
> """
> The same command finished successfully on master. Any ideas?

Fixed this problem for commenting "#import nemo".

File: docs/docs_zh/sources/source/asr/tutorial.rst [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+--------
+我们会使用 LibriSpeech :cite:`panayotov2015librispeech` 数据集。下面这些脚本会下载并且把 Librispeech 转成 `nemo_asr` 需要的数据格式：
+
+.. code-block:: bash
+
+    mkdir data
+    # 我们需要安装 sox
+    # 在 ubuntu 上安装 sox, 只需要：sudo apt-get install sox
+    # 接着：pip install sox
+    # get_librispeech_data.py script is located under <nemo_git_repo_root>/scripts
```
[the remark this answers is not in the document]
Comment (Slyne) [reply docs/docs_zh/sources/source/asr/tutorial.rst]: updated

**Comment by redoctopus** [comment]
Just figured out how to pull and build, and I get some warnings:
```
/home/jocelynh/Desktop/hub_nemo/docs/docs_zh/sources/source/asr/tutorial.rst:308: WARNING: Title underline too short.

用 KenLM 构建的语言模型
~~~~~~~~~~~~~~~~~~~~~~
WARNING: autodoc: failed to import module 'data' from module 'nemo.backends.pytorch.common'; the following exception was raised:
No module named 'nemo.backends.pytorch.common.data'
/home/jocelynh/Desktop/hub_nemo/docs/docs_zh/sources/source/nlp/joint_intent_slot_filling.rst:2: WARNING: Duplicate explicit target name: "这里".
/home/jocelynh/Desktop/hub_nemo/docs/docs_zh/sources/source/nlp/neural_machine_translation.rst:2: WARNING: Duplicate explicit target name: "这里".
/home/jocelynh/Desktop/hub_nemo/docs/docs_zh/sources/source/tts/datasets.rst:2: WARNING: Duplicate explicit target name: "ljspeech".
/home/jocelynh/Desktop/hub_nemo/docs/docs_zh/sources/source/tts/datasets.rst:2: WARNING: Duplicate explicit target name: "辅助脚本".
/home/jocelynh/Desktop/hub_nemo/docs/docs_zh/sources/source/tts/tutorial.rst:35: WARNING: undefined label: ljspeech (if the link has no caption the label must precede a section header)
```
Could you guys fix these?

**Pull request merged**